docs/user_guide/libraries.rst

```diff
@@ -40,22 +44,22 @@ Similarly to :ref:`PDKs<PDKs>`, library modules must implement the following fun
 
    * - Function
      - Description
-     - Arg
+     - Args
      - Returns
      - Used by
      - Required
 
    * - **setup**
      - Library setup function
-     - chip
-     - chip
+     - chip, lib_name
+     - lib
      - :meth:`.load_lib()`
      - yes
 
    * - **make_docs**
      - Doc generator
      - None
```

Comment:
Chip

Reply:
Looks like we don't need lib_name, but the Library setup functions are supposed to return a Library, not a Chip.

Reply:
I see these weren't clear, the make_docs Args is Chip